### ISSUE TITLE
Accessibility - Video Player - Keyboard - Focus is not visible for Video Slider

### ISSUE THREAD

**Issue by Praveen-sr** [carbon-design-system/ibm-dotcom-library#1775]
## Detailed description

Issue :- 
Focus is going to the slider but it is not visible at all.

Expected -

Expected Result:
Provide thick and strong visual focus
1. If, Visual Focus Indicator with 3 CSS px stroke then provide a sufficient contrast that is equal to 3:1. 
2. Visual focus indicator for a link that is a 1 CSS px then provide a sufficient contrast greater than 4.5:1

For more information please find the below link,
https://www.w3.org/WAI/GL/low-vision-a11y-tf/wiki/Contrast_(Minimum)

**Comment by RobertaJHahn** [comment]
Moving this issue to sprint 20-08.  The dev work has to be done during the first week of the sprint to include it in release v1.6.0.

**Comment by RaphaelAmadeu** [comment]
This accessibility issue is happening inside the videoplayer, that is provided by Kaltura, so we can't edit it on our end.

Thanks!
CC: @RobertaJHahn  @jeffchew

**Comment by ljcarot** [comment]
So we should remove Bug label and severity, then close.

**Comment by ljcarot** [comment]
Going to close this issue.  Will send details to video team to discuss with the Kaltura product owner.
